modules/magniteAnalyticsAdapter.js

```diff
@@ -723,9 +729,11 @@ const getLatencies = (args, auctionStart) => {
 }
 
 let browser;
+let auctionCount = 0;
```

Comment:
can you move this and other variable declarations to the top of the file? They are "global" to the file and used everywhere and it's strange that they're defined at line 732.